app/controllers/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find(params[:id])
+    if @user.nil?
+      render json: {message: "Cannot find user"}, status: :not_found
+    end
+    @info = {user: @user,services: Service.where(user_id: params[:id]),orders: Order.where(user_id: params[:id])}
```

Comment:
You should have setup your models so this will work:

```rb
@services = current_user.services
@orders = current_user.orders
@info = {user: @user, services: @services, orders: @orders}
```